### PR TITLE
Pushed revisions for the settings icons

### PR DESCRIPTION
**Pull request by edokoa**
As discussed in: https://github.com/github/primer/issues/486

Icons are still pending of discussion with the whole team but are a blocker right now. We will review again later.

image

https://www.figma.com/file/2rjsQp42uKvzPXzLZIqOeA/Octicons-v2---Source?node-id=7071%3A51354

Mock of a menu with all the icons

image

### CODE REVIEW

**Review by edokoa — COMMENTED** [review]
Checked transparent backgrounds

**Comment by edokoa** [comment]
cc @alliethu and @ichelsea as this includes retouching the accessibility icon.

**Review by vdepizzol — APPROVED** [review]
✨ ✨ ✨

File: icons/log-16.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="17" viewBox="0 0 16 17"><path d="M5 9.25a.75.75 0 01.75-.75h4a.75.75 0 010 1.5h-4A.75.75 0 015 9.25zM4 11.5A.75.75 0 004 13h4a.75.75 0 000-1.5H4z"/><path fill-rule="evenodd" d="M13 .995H3a3 3 0 00-3 3c0 .676.224 1.254.603 1.722.526.65 1.331.783 1.907.783h1.177c-.364.662-.814 1.339-1.287 2.048-.205.309-.414.624-.623.946C.891 10.865 0 12.418 0 14a3 3 0 003 3h10a3 3 0 001.667-5.494.75.75 0 00-.834 1.246A1.5 1.5 0 1111.5 14c0-.642.225-1.347.623-2.136.397-.787.933-1.593 1.501-2.446l.011-.017c.554-.83 1.139-1.709 1.582-2.588.445-.885.783-1.836.783-2.818 0-1.672-1.346-3-3-3zm-10 1.5a1.5 1.5 0 00-1.5 1.5c0 .321.1.569.27.778.097.12.325.227.74.227h7.674A2.737 2.737 0 0110 3.995c0-.546.146-1.059.401-1.5H3zm10 0c.831 0 1.5.662 1.5 1.5 0 .646-.225 1.353-.623 2.143-.398.79-.933 1.595-1.501 2.448l-.017.026c-.552.828-1.134 1.702-1.575 2.576C10.338 12.072 10 13.021 10 14c0 .546.146 1.059.401 1.5H3A1.5 1.5 0 011.5 14c0-1.084.63-2.289 1.537-3.692.177-.274.366-.556.558-.845.632-.948 1.306-1.96 1.773-2.963h6.382a.75.75 0 00.417-1.373c-.444-.298-.667-.656-.667-1.132a1.5 1.5 0 011.5-1.5z"/></svg>
```
Comment (vdepizzol) [inline icons/log-16.svg:1]: @edokoa this file is using a height of `17` in the `height` and `viewBox` attributes.

**Review by ichelsea — APPROVED** [review]
I'm comfortable with where the a11y icon has landed! What's odd for me is it's different than the accessibility icon than I'm used to seeing but I think it still represents what we're going for and it's cute! I don't have any recommendations for what to do differently so I think this icon is in a good spot to go live.

File: icons/log-16.svg (hunk 1 above)
Comment (edokoa) [reply icons/log-16.svg:1]: I saw Figma doing something strange with another icon and trying to push it as 15.99999999999999999 even if it was 16 in the interface. I'll re-upload  this one. Thanks for catching this.

**Comment by emilybrick** [comment]
The accessibility icon is reading to me as a stick figure or illustration of a stick figure more than it is a representation for accessibility. Can we keep working on this one without blocking the others? think it may need another pass.

**Comment by tallys** [comment]
Hi @edokoa ! Asked some accessibility folks about the icons - we will see what it is. My initial take was reading it as more "profile" and not "accessibility. Perhaps for me what signals it being accessibility is a T shaped stick figure (not arms down) and probably standing straight up and not having the A-frame shape. Overall scroll is getting positive feedback, but I'm not too familiar with the history of scrolls at GitHub :)

**Comment by adrianmg** [comment]
Thanks for sharing, Javi!! Dropping some feedback regarding **Pages icon**:

- I'm finding its shape too close and similar to the `copy-to-clipboard` one.
- I wonder if we could explore a different concept. At the moment, it feels like a direct translation from `GitHub Pages` to `Pages from a document` (or pages from the wiki) that doesn't resonate much with the service itself.

image

File: icons/log-16.svg (hunk 1 above)
Comment (edokoa) [reply icons/log-16.svg]: Sorry for not updating here, but this was corrected in https://github.com/primer/octicons/pull/715/commits/f48e121904a8f8e4c49dc37ef2e5d08ac63088fe

**Comment by ashygee** [comment]
> I wonder if we could explore a different concept. At the moment, it feels like a direct translation from GitHub Pages to Pages from a document (or files/wiki) that doesn't resonate much with the service itself.

@adrianmg thank you for pointing that out. We are going to move forward with recommending the current [`browser` icon](https://primer.style/octicons/browser-16) to be used for the GitHub Pages link.

We spoke with @samoshin from our Brand team and we will move forward with the current logs icon (scroll) for now with room to iterate at a later time. 

For the accessibility icon we're going to move with the current proposal as there was no definitive feedback that there was anything conflicting with this version. This is the first iteration of the accessibility icon which is needed within Octicons and we can iterate later if we formulate more of an opinion as to how we want to visualize accessilibity.

**Comment by edokoa** [comment]
I'm going to push this one. I feel the arms are 1px too low in the previous version.

image

[Figma](https://www.figma.com/file/2rjsQp42uKvzPXzLZIqOeA/Octicons-v2---Source?node-id=7110%3A52119)

**Review by ashygee — APPROVED** [review]

**Pull request merged**